admin-manual/installation-setup/installation/installation.rst

```diff
@@ -20,19 +20,19 @@ Archivematica is not a single application - dozens of different components and
 tools are required for a fully working installation. As a result, there are many
 possible deployment configurations.
 
-These instructions are designed to get you up and running as quickly as
-possible, even if you are not familiar the various tools and applications that
+The following table lists all help topics on installing Archivematica for various intended uses and host environments. The related document links are designed to get you up and running as quickly as
```

Comment:
118 is the best I can do to tighten the intro.

Reply:
Oh sorry! All I meant is that there should be a line break every 80 characters. You can write as much as you want :)

So instead of 

```
The following table lists all help topics on installing Archivematica for various intended uses and host environments. The related document links are designed to get you up and running as quickly as
```

it should be 

```
The following table lists all help topics on installing Archivematica for
various intended uses and host environments. The related document links are
designed to get you up and running as quickly as
```

If you use Atom.io as your text editor, I find [atom-hard-wrap](https://github.com/forivall/atom-hard-wrap) helpful!